pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -91,16 +91,13 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 		wc.Resources.MemoryLimitInBytes = memoryLimit
 	}
 
-	// setup security context
-	effectiveSc := securitycontext.DetermineEffectiveSecurityContext(pod, container)
-	// RunAsUser only supports int64 from Kubernetes API, but Windows containers only support username.
-	if effectiveSc.RunAsUser != nil {
-		return nil, fmt.Errorf("run as uid (%d) is not supported on Windows", *effectiveSc.RunAsUser)
+	if container.SecurityContext != nil {
```

Comment:
Is there anything in security context that works on windows? Is there any way to validate this at API validation time? Pods failing to start on bad config is not the best UX.

Reply:
> Is there anything in security context that works on windows? 
At this time, no: https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20190103-windows-node-support.md#v1podsecuritycontext

> Is there any way to validate this at API validation time? Pods failing to start on bad config is not the best UX.
I'm not sure, I'll have to look into API validation to see if we can introduce something

Reply:
It is debatable, but I'd say yes, depending on their values, but mostly because they're basically noop. For example: if ``readOnlyFilesystem=false``, or ``runAsNonRoot = false``.